apps/etterlatte-fordeler/src/main/kotlin/fordeler/FordelerService.kt

```diff
@@ -13,39 +13,75 @@ import java.time.Clock
 import java.time.OffsetDateTime
 import java.time.format.DateTimeFormatter
 
+enum class Vedtaksloesning {
+    DOFFEN, PESYS
+}
+data class Fordeling(val soeknadId: Long, val fordeltTil: Vedtaksloesning, val kriterier: List<FordelerKriterie>)
+
 sealed class FordelerResultat {
     object GyldigForBehandling : FordelerResultat()
     class UgyldigHendelse(val message: String) : FordelerResultat()
     class IkkeGyldigForBehandling(val ikkeOppfylteKriterier: List<FordelerKriterie>) : FordelerResultat()
 }
 
+fun featureFromEnv(featureName: String): Long {
+    return (System.getenv()[featureName]?.toLong() ?: 0).takeIf { it > -1 } ?: Long.MAX_VALUE
```

Comment:
Nå henter vi ut env to steder, også i main, kan vi sende den ned hit og bruke den samme her?
slik at vi får `environment[featureName]?....`

Reply:
+1 på den - gjør det også mer fleksibelt ift testing 👍